File: compiler/tests/perf_fib.go
package main

import "fmt"

func fib(n int32) int32 {
	if n <= 1 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	result := fib(42)
	_ = result
	fmt.Println("fib done")
}

File: compiler/tests/perf_fib.rs
fn fib(n: i32) -> i32 {
    if n <= 1 { return n; }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let result = fib(42);
    let _ = result;
    println!("fib done");
}
